c_common/models/live_packet_gather/src/live_packet_gather.c

```diff
@@ -184,6 +187,9 @@ static void timer_callback(uint unused0, uint unused1) {
     time++;
     log_debug("Timer tick %u", time);
 
+    // Reset the count of packets sent in the current timestep
+    packets_sent = 0;
```

Comment:
This fixes a bug where setting `config.packets_per_timestamp` to non-zero would result in that being the maximum total number of packets ever sent, which was a dumb and non-useful feature! The problem was that the counter (`packets_sent`) was never being reset at the start of the next timestep.

Reply:
guessing thats a legacy debugging tool